src/Drivers/Calendar/GoogleCalendarDriver.php

```
@@ -0,0 +1,1093 @@
+     *
+     * @param  CalendarWatchChannel  $channel  The registration to replace.
+     * @param  string  $callbackUrl  Where Google should post notifications.
+     *
+     * @throws CalendarSyncException When Google refuses to register the new
+     *                               channel or is unreachable.
+     *
+     * @return CalendarWatchChannel The renewed registration.
+     */
+    public function renewSubscription( CalendarWatchChannel $channel, string $callbackUrl ): CalendarWatchChannel
+    {
+        $connection = $channel->connection;
```

PHPStan (2.2.7)

[error] Access to protected property ArtisanPackUI\Bookings\Models\CalendarWatchChannel::$connection.


(property.protected)